src/tools/compiletest/src/header/tests.rs

```diff
@@ -380,7 +380,7 @@ fn profiler_runtime() {
 #[test]
 fn asm_support() {
     let asms = [
-        ("avr-unknown-gnu-atmega328", false),
+        ("avr-unknown-unknown", false),
```

Comment:
The problem is this line, PR CI runs compiletest tool tests at `--stage 0`, which uses a prebuilt beta rustc. This beta rustc does not know about `avr-unknown-unknown` because it doesn't exist to beta rustc, it only knows about `avr-unknown-gnu-atmega328`. You need to guard this with something like

```rs
#[cfg(bootstrap)]
("avr-unknown-gnu-atmega328", false),
#[cfg(not(bootstrap))]
("avr-unknown-unknown", false),
```

Then T-release should delete the `#[cfg(bootstrap)]`'d `avr-unknown-gnu-atmega328` next bump and when a beta rustc knowns about the renamed `avr-unknown-unknown` target delete `#[cfg(not(bootstrap))]` attribute.

Reply:
Cool, let's try it 🙂